sway/commands/output/toggle.c

```diff
@@ -0,0 +1,37 @@
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/output.h"
+
+struct cmd_results *output_cmd_toggle(int argc, char **argv) {
+    if (!config->handler_context.output_config) {
+        return cmd_results_new(CMD_FAILURE, "Missing output config");
+    }
+
+    struct output_config *output = config->handler_context.output_config;
+
+    if (argc > 1 && strcmp(output->name, "*") == 0) {
```

Comment:
What is the reasoning for `argc > 1` here? This shouldn't take any arguments

Reply:
I saw a similar line in `transform.c`, I guess that was for something else.

Reply:
`transform` takes arguments. this handler does not